openstack/elektra/charts/postgres/templates/backup-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.backup.enabled }}
```

Comment:
Ok, but why specifically? To me it seems arbitrary that 2 of 11 config options are put in in a seperate file. This requires an additional k8s spec (configmap) and the deployment is more complex (because of the mounting). I think with helm there are way fewer occasions where a configmap is really needed, as the deployment is a template. The only reason I can see for adding a configmap is if a software requires a configfile and can't be configured via environment variables.